### ISSUE TITLE
[Docs]: Valle_v2 pretrained model info

### ISSUE THREAD

**Issue by acul3** [open-mmlab/Amphion#259]
hey @jiaqili3 or anyone involve

do you have information about how many epoch your valle_v2 train ?

[valle_ar_mls_196000.bin](https://huggingface.co/amphion/valle/blob/main/valle_ar_mls_196000.bin) 

[valle_nar_mls_164000.bin](https://huggingface.co/amphion/valle/blob/main/valle_nar_mls_164000.bin)

( i assume 196000, 164000 is step) ?

a tensorboard loss could also be helpfull, thank you

**Comment by jiaqili3** [comment]
Hi @acul3, yes the 190k and 160k are steps, and each model has been trained on about 1 epoch. For the loss curve, it's  a bit hard to find, but we'll post them for future model releases. Thanks!

**Comment by acul3** [comment]
thank for the reply and confirmation
